lib/WsdlParser.js

```diff
@@ -235,7 +236,7 @@ class WsdlParser {
         wsdlOperation.xpathInfo = this.informationService.getOperationxPathInfo(
```

Comment:
Ensure that there is safe handling in this method when the argument is `null`. 2nd order effects 😄